pachinko/time_period_step_agent.py

```diff
@@ -61,37 +61,45 @@ def act(self, previous_observation: np.ndarray) -> Tuple[Any, dict]:
         return action, {}
 
 
-class Periodic:
-    """Agent that maintains and optimises on separate conversion rates for each step in period."""
+# A list containing a tuple of (num_successes, timestep) for each time an action was picked,
+# representing the full history of that action's conversions
+ConversionHistory = List[Tuple[int, int]]
```

Comment:
Enhancement: would it be worth defining types like: 
```
NumberSuccesses = int
Timestep = int
```
to enable 
```
ConversionHistory = List[Tuple[NumberSuccesses, Timestep]]
```
Given the importance of that variable for the agents etc., I found myself having to refer back to the commend on line 64 repeatedly, so I kept forgetting which was with the 2uple. Having it in the actual type def instead of having to stitch that with the comment would help a lot I think. 

Thoughts?